charts/prometheus-thanos/values.yaml

```diff
@@ -116,6 +116,8 @@ storeGateway:
     config:
       max_size: 500MB
   chunkPoolSize: 500MB
+  binaryIndexHeader:
+    enable: false
```

Comment:
please use "enabled" to be consitent with other optional stuff.

Reply:
Thanks, updated and rebased on master.